src/Manifest.cc

```diff
@@ -420,20 +420,33 @@ validateDepName(const StringRef name) {
     throw PoacError("dependency name must end with an alphanumeric character");
```

Comment:
You should update the above comments as well, i.e., Trailing `-`, `_`, `/`, and `.` are not allowed.